features/ftue/impl/src/main/kotlin/io/element/android/features/ftue/impl/notifications/NotificationsOptInView.kt

```diff
@@ -0,0 +1,206 @@
+import io.element.android.libraries.theme.ElementTheme
+import io.element.android.libraries.ui.strings.CommonStrings
+
+@Composable
+fun NotificationsOptInView(
+    state: NotificationsOptInState,
+    onBack: () -> Unit,
+    modifier: Modifier = Modifier,
+) {
+    BackHandler(onBack = onBack)
+
+    if (state.notificationsPermissionState.permissionAlreadyDenied) {
+        LaunchedEffect(Unit) {
+            state.eventSink(NotificationsOptInEvents.NotNowClicked)
+        }
+    }
+
+    HeaderFooterPage(
+        modifier = modifier
+            .systemBarsPadding()
+            .fillMaxSize(),
+        header = { NotificationsOptInHeader(modifier = Modifier.padding(top = 60.dp, bottom = 12.dp),) },
+        footer = { NotificationsOptInFooter(state) },
+    ) {
+        NotificationsOptInContent(modifier = Modifier.fillMaxWidth())
+    }
+}
+
+@Composable
+private fun NotificationsOptInHeader(
+    modifier: Modifier = Modifier,
+) {
+    IconTitleSubtitleMolecule(
+        modifier = modifier,
+        title = "Allow notifications and never miss a message",
+        subTitle = "You can change your settings later.",
+        iconImageVector = Icons.Default.Notifications,
+    )
+}
+
+@Composable
+private fun NotificationsOptInFooter(state: NotificationsOptInState) {
+    ButtonColumnMolecule {
+        Button(
+            modifier = Modifier.fillMaxWidth(),
+            text = stringResource(CommonStrings.action_ok),
+            onClick = {
+                state.eventSink(NotificationsOptInEvents.ContinueClicked)
+            }
+        )
+        TextButton(
+            modifier = Modifier.fillMaxWidth(),
+            text = stringResource(CommonStrings.action_not_now),
+            onClick = {
+                state.eventSink(NotificationsOptInEvents.NotNowClicked)
+            }
+        )
+    }
+}
+
+@Composable
+private fun NotificationsOptInContent(
+    modifier: Modifier = Modifier,
+) {
+    Box(modifier = modifier.fillMaxSize(), contentAlignment = Alignment.Center) {
+        Column(
+            verticalArrangement = Arrangement.spacedBy(
+                16.dp,
+                alignment = Alignment.CenterVertically
+            )
+        ) {
+            NotificationRow(
+                avatarLetter = "M",
+                avatarBackgroundColor = Color(0xFFE3F5F8),
+                avatarForegroundColor = Color(0xFF004077),
+                firstRowPercent = 1f,
+                secondRowPercent = 0.4f
+            )
+
+            NotificationRow(
+                avatarLetter = "A",
+                avatarBackgroundColor = Color(0xFFFAEEFB),
+                avatarForegroundColor = Color(0xFF671481),
+                firstRowPercent = 1f,
+                secondRowPercent = 1f
+            )
+
+            NotificationRow(
+                avatarLetter = "T",
+                avatarBackgroundColor = Color(0xFFFFEFE4),
+                avatarForegroundColor = Color(0xFF850000),
```

Comment:
I guess we could use color defined in https://github.com/vector-im/element-x-android/pull/1224 instead of hard-coded color, but not a big deal.

We will also have different color for light and dark theme, which is better I think.

(even if those colors are used in the timeline and not in the Notifications (yet?), but that's just cosmetics...)

Reply:
I think we should merge your avatars iteration PR first and them maybe reuse an actual `Avatar` here with the right colors. What do you think?